CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 
 - .NET only, warning in logs about End of Support date
   and upcoming End of Support date for .NET version.
+- ARM64 support for Ubuntu, Alpine and Debian based images.
+- ARM64 support for the `OpenTelemetry.AutoInstrumentation` NuGet package.
```

Comment:
I would consider putting something like - experimental support for (here and in the docs). With this, we can release ARM64 in stable packages. Without this, it should be supported only on prereleases. It applies also to REDME.md

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3277/commits/33c5bf20fe54381f78566fe8b0b74f13be8b8915